src/context/__tests__/Auth.tsx

```diff
@@ -16,7 +16,11 @@ const userMissing = 'No User Found';
 const FakeComponent = () => {
     const { user } = useAuth();
 
-    return user ? user : userMissing;
+    if (user) {
```

Comment:
Could this not be written with a single return statement?

Reply:
Yeah... I was getting frustrated with this one and was trying different things and just landed on this and stopped here. I'll fix it with my next PR.